### PR TITLE
add authz helpers

### PR DESCRIPTION
**Pull request by mojiiba**
This pull add helper to populate rpc paths related to fp tests to simplify testing authz.

### CODE REVIEW

File: internal/security/authz/authz.go [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+	// rpc for which the rule is specified
+	Request struct {
+		Paths []string `json:"paths"`
+	} `json:"request"`
+}
+
+func createRule(users []string, rpcs []*gnxi.RPC) Rule {
+	rule := Rule{}
+	for _, rpc := range rpcs {
+		rule.Name = rule.Name + rpc.FQN
```
Comment (harishkumar3091) [inline internal/security/authz/authz.go:66]: I believe this is QFN perhaps ?

**Comment by xw-g** [comment]
I think this PR can be closed. https://github.com/openconfig/featureprofiles/pull/2136 includes all the changes of this PR. And I have already reviewed #2136

File: internal/security/gnxi/rpcs.go [hunk 2]
```diff
@@ -0,0 +1,1078 @@
+// This following provide a list of all RPCs related to FP testing.
+// The below code is generated using ../gen/generate.go. Please do not modify.
+package gnxi
+
+type rpcs struct {
+	ALL                                                                   *RPC
```
Comment (greg-dennis) [inline internal/security/gnxi/rpcs.go:6]: Go style is to _not_ put these in all caps. Also, do the fields need to be exported here?

File: internal/security/gnxi/rpcexec.go [hunk 3]
```diff
@@ -0,0 +1,665 @@
...
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ygnmi/ygnmi"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	gpb "github.com/openconfig/gnmi/proto/gnmi"
+)
+
+// function AllRPC implements a sample request for service * to validate if authz works as expected.
```
Comment (greg-dennis) [inline internal/security/gnxi/rpcexec.go:21]: function comments should start with the name of the function, e.g. "AllRPC implements ..."

**Review by greg-dennis — CHANGES_REQUESTED** [review]

**Pull request closed without merging**